controllers/front/return.php

```diff
@@ -111,12 +124,25 @@ public function postProcess()
                         'moduleId' => $moduleId,
                         'secureKey' => $secureKey,
                         'selectedCard' => $selectedCard,
-                    ]
+                    ],
+                    true
                 ));
+                exit; // Ensure the script stops after redirect
```

Comment:
exit is not needed https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/classes/Tools.php#L223